collect_app/src/main/java/org/odk/collect/android/widgets/SelectWidget.java

```diff
@@ -126,7 +126,26 @@ public void onCompletion(MediaPlayer mediaPlayer) {
         }
     }
 
-    protected MediaLayout createMediaLayout(int index, TextView textView) {
+    public MediaLayout createMediaLayout(int index, TextView textView) {
+        MediaLayout mediaLayout = new MediaLayout(getContext());
+
+        initMediaLayoutSetUp(mediaLayout);
+        setUpAVT(mediaLayout, index, textView);
+
+        if (index != items.size() - 1) {
+            mediaLayout.addDivider();
+        }
+
+        return mediaLayout;
+    }
+
+    public void initMediaLayoutSetUp(MediaLayout mediaLayout) {
+        mediaLayout.setAudioListener(this);
+        mediaLayout.setPlayTextColor(getPlayColor());
+        playList.add(mediaLayout);
+    }
+
+    public void setUpAVT(MediaLayout mediaLayout, int index, TextView textView) {
```

Comment:
I know `AVT` is used in the `setAVT` method but it's not very clear what it means (Audio-Video-TextView?). Since this is a new method being created, would be great to give it a really descriptive name. I guess `setUpAudioVideoText` might be ok.

Reply:
Ok done.